medcat/cat.py

```diff
@@ -309,7 +309,6 @@ def __call__(self, text: Optional[str], do_train: bool = False) -> Optional[Doc]
             return None
         else:
             text = self._get_trimmed_text(str(text))
```

Comment:
not sure we need the Optional type hint and therefore the cast here. should this just be def __call__(self, text: str .. )

Reply:
It is for backward compatibility. AFAIK, the CAT callable has been able to take in None and raise no exceptions since day one.

Reply:
fair enough - actually yes this rings a bell for MedCATservice integration / running over multiple docs, sometimes there  `None` texts... annoying but fine makes sense to keep